app/views/games/_chessboard.html.erb

```diff
@@ -1,3 +1,6 @@
+  <h3 class="alert alert-info text-center">
+    <%= "White Player's Turn" %>
```

Comment:
Not super important, but this doesn't have to be ERB, since it's just HTML.  However you likely want to have Rails pre-render the current player's info, so if you use a `currently_black_turn?` method like from below:

```html
<h3 class="alert alert-info text-center">
  <% if @game.currently_black_turn? %>
    Black Player's Turn
  <% else %>
    White Player's Turn
  <% end %>
</h3>
```